book/analysis/introduction.md

```diff
@@ -110,20 +94,26 @@ Sum is 500000500000, required  0.1729250 seconds
 Sum is 500000500000, required  0.1646299 seconds
 ```
 
-In this case, the average again turns out to be about 10 times the
-previous.
+Not only does the pattern still hold, but there appears to be a correlation
+between the _size_ of the input and the _time_ needed to process that input.
+Every time we increase `n` by some factor, the run time increases by the same
+factor. Nasty; I certainly don't have time to wait around for the sum of the
```

Comment:
@aleksharma12 "Nasty" feels a little risque. Also in practice, linear running time is usually a great result :). We may want to build that intuition by just dropping the "nasty".